src/java.desktop/macosx/native/libawt_lwawt/awt/CPrinterJob.m

```diff
@@ -423,6 +429,19 @@ static void javaPrinterJobToNSPrintInfo(JNIEnv* env, jobject srcPrinterJob, jobj
     if (page != NULL) {
         javaPageFormatToNSPrintInfo(env, NULL, page, dst);
     }
+
+    jstring dest = JNFCallObjectMethod(env, srcPrinterJob, jm_getDestinationFile);
+    if (dest != NULL) {
+       NSString *nsDestStr = JNFJavaToNSString(env, dest);
+       [dst setJobDisposition:NSPrintSaveJob];
+      // [printingDictionary setObject:nsDestStr forKey:NSPrintSavePath];
+       //NSURL *nsURL = [NSURL URLWithString:nsDestStr];
+       NSURL *nsURL = [NSURL fileURLWithPath:nsDestStr isDirectory:NO];
+       [printingDictionary setObject:nsURL forKey:NSPrintJobSavingURL];
+//        JNFCallVoidMethod(env, dstPrinterJob, jm_setPrintToFile, true);
```

Comment:
Does it look like the commented lines are not needed?

Reply:
Hmm. I could swear I removed those and re-ordered the first two lines as well.